cmd/strided/config/config.go

```diff
@@ -23,12 +26,23 @@ const (
 )
 
 const (
-	// DisplayDenom defines the denomination displayed to users in client applications.
-	DisplayDenom = "Stride"
-	// StrideDenom defines to the default denomination used in Stride (staking, EVN, governance, etc)
-	StrideDenom = "strd"
+	HumanCoinUnit    = "Stride"
+	BaseCoinUnit     = "strd"
+	DefaultBondDenom = BaseCoinUnit
 )
 
```

Comment:
do we need these setup changes? i thought the issue we were running into was somewhere else?

i'm not opposed to these changes though, probably fine to keep them in

Reply:
Agreed they're not the main focus here but I think we should make em, and they pertain to token things so they make sense for this PR !